### PR TITLE
Read data from SQL query results

### PR DESCRIPTION
**Pull request by sophieball**
Changes were necessary because G will pull data from SQL database directly, not from disk. Also, I changed the column `toxic` to the more general term `label`, because G's outcome variable is `push-back` not exactly `toxicity`.

### CODE REVIEW

File: src/convo_word_freq_diff.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Lint as: python3
```
Comment (CaptainEmerson) [inline src/convo_word_freq_diff.py:1]: this file should not be included, right?

File: src/train_classifier_g.py [hunk 2]
```diff
@@ -6,16 +11,23 @@
...
 
   print("Loading training data.")
-  train_dat = pd.read_csv(join(path, "src/data/training_data.csv"))
+  # executes a SQL query and returns a pd.DataFrame
+  # 3 columns: _id, text, label(0/1)
+  train_dat = pd.read_sql_query("SQL_QUERY_FOR_TRANING_DATA")
   s.set_train_set(train_dat)
-  test_dat = pd.read_csv(join(path, "src/data/test_data.csv"))
+  # 2 columns: _id, text
+  test_dat = pd.read_sql_query("SQL_QUERY_FOR_TEST_DATA")
```
Comment (CaptainEmerson) [inline src/train_classifier_g.py:26]: Two requests:
* Abstract this a bit. Add a new function, likely in another file, like retrieve_data(), which you call here. In that function, you can have your current read_csv call. I will then change that and only that file when running it with my data. This'll centralize my changes more and reduce the likelihood of a merge conflict.
* Choose the train/test split here for me. 70/30 is probably fine for now. I don't have any preconcieved notions of how I want my data split, and I see no difference in splitting it differently from how you do for the CSV.

File: src/util.py [hunk 3]
```diff
@@ -141,14 +141,14 @@ def prepare_dataset(train_data,map_toxic=True):
 
     train_data["text"] = train_data["text"].map(str)
     train_data["original_text"] = train_data["text"]
-    train_data["uppercase"] = train_data["text"].map(percent_uppercase)
+    train_data["uppercase"] = train_data["text"].map(text_modifier.percent_uppercase)
```
Comment (CaptainEmerson) [inline src/util.py:144]: what's going on here with text_modifier?

File: src/convo_word_freq_diff.py (hunk 1 above)
Comment (sophieball) [reply src/convo_word_freq_diff.py:1]: Right.. that was the carry-over from previous pr...

File: src/train_classifier_g.py (hunk 2 above)
Comment (sophieball) [reply src/train_classifier_g.py:26]: I think there's a misnomer here.. test_data isn't really used to test the performance of the model; it's the dataset you want to apply the model to. Train/test split and cross-validation is in suite.py.

File: src/util.py (hunk 3 above)
Comment (sophieball) [reply src/util.py:144]: This function is defined in text_modifier.py. I added the module name here so that I can easily tell which file it comes from..

**Pull request closed without merging**